src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -66,7 +67,9 @@ public ReadOnlyPerson getPerson() {
     public CommandResult execute() {
         try {
             addressBook.addPerson(toAdd);
-            return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+            List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+            System.out.println(String.format(MESSAGE_SUCCESS, toAdd) + "\n\nUpdated Address Book:");
```

Comment:
Although this code gets the job done, it violates certain principles of software engineering. Try to refactor the code to ensure [SLAP](https://nuscs2113-ay1819s2.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard) and [SRP](https://nuscs2113-ay1819s2.github.io/website/se-book-adapted/chapters/principles.html#single-responsibility-principle) is maintained.